packages/camera/camera/example/integration_test/camera_test.dart

```diff
@@ -92,7 +92,9 @@ void main() {
         await controller.dispose();
       }
     }
-  }, skip: !Platform.isAndroid);
+  },
+      skip:
+          true); // TODO(egarciad): Fix https://github.com/flutter/flutter/issues/93686.
```

Comment:
Optional nit for these: if you put the comment above the line rather than at the end of it, you can avoid the weird breaking between `skip` and `true`; that's how we've commented other similar cases.

Reply:
done